lib/search.js

```diff
@@ -4,10 +4,11 @@ var cheerio = require('cheerio');
 
 var h = require('./helpers');
 
+var optsGlobal = null;
```

Comment:
actually I think you can just put it in the query object to be consistent with the rest of the code